03/app.js

```diff
@@ -0,0 +1,13 @@
+const buttons = document.querySelectorAll('button');
+
+if(buttons) {
```

Comment:
Natomiast tutaj warunek jest zbędny.
`.querySelectoAll` zawsze zwraca tablicę (pustą lub nie), a pusta tablica to zawsze prawda więc warunek nic nam nie daje.
Tutaj warunku nie trzeba bo jak mamy pustą tablicę to po prostu `.forEach` się nie uruchomi :)